src/config/config.cc

```diff
@@ -859,6 +853,11 @@ Status Config::Set(Server *srv, std::string key, const std::string &value) {
   return Status::OK();
 }
 
+bool Config::ignoreIfNotModified(const std::string &key, const std::string &value) const {
+  auto iter = fields_.find(key);
+  return iter != fields_.end() && iter->second->Default() == value;
+}
```

Comment:
From the semantics of this function, I think it's better to be named as `checkOptionIsDefault` or something like that since it has no operation about "ignore".

Reply:
Longer option from me: `isFieldValueSetToDefault` :)

Reply:
ok, makes sense.

Reply:
I guess we can use `checkFieldValueIsDefault`